src/panel/category_panel.js

```diff
@@ -82,6 +82,23 @@ export default class CategoryPanel {
     this.search()
     await this.panel.update()
     UrlState.pushUrl()
+
+    // Apply minimal zoom when opening a category
+
+    // Zoom < 5: focus on Paris
+    if(window.map.mb.getZoom() < 5){
```

Comment:
Wouldn't a `switch` be more appropriate in here? It'd allow to call `window.map.mb.getZoom()` only once.

Reply:
I agree to calling getZoom only once, but not for the switch. Switches are not ideal to test < / > conditions...  right?